fetch.bs

```diff
@@ -5483,6 +5483,9 @@ constructor must run these steps:
 
    <dt><a for=request>history-navigation flag</a>
    <dd><var>request</var>'s <a for=request>history-navigation flag</a>.
+
+   <dt><a for=request>priority</a>
+   <dd><var>request</var>'s <a for=request>priority</a>.
```

Comment:
Ideally we keep these in a similar order as the order they are introduced in, so this should be higher up. Otherwise I think this change looks fine.